src/BookReader/DragScrollable.js

```diff
@@ -0,0 +1,233 @@
+// @ts-check
+/*
+ * jQuery dragscrollable Plugin
```

Comment:
I was thinking of that, but there are a few places where it would require a bit too much tweaking :/ The MouseEvent jquery wrappers, and the use of jquery event namespaces for detaching event listeners. But definitely in the realm of feasibility, just yeah, out of scope for this one.